app/components/dashboard/run/left-panel/component.js

```diff
@@ -165,5 +166,23 @@ export default Component.extend(FullScreenMixin, {
         denyDelete() {
             return true;
         },
+        
+        exportZip(id) {
+          const token = this.get('tokenHandler').getWholeTaleAuthToken();
+          let url = `${config.apiUrl}/tale/${id}/export`;
+          window.open(url + '?token=' + token + '&taleFormat=native', '_newtab');
```

Comment:
I don't think `_newtab` is a valid value for the `target` attribute... I think `_blank` should open in a new tab/window (based on the user's browser preferences), if that's the desired behavior

See:
* https://www.w3schools.com/jsref/met_win_open.asp
* https://www.w3schools.com/tags/att_a_target.asp
* https://stackoverflow.com/questions/4964130/target-blank-vs-target-new/8867079#8867079
* https://stackoverflow.com/questions/18360438/attribute-target-newtab-for-links